bin/peas_shell

```diff
@@ -86,6 +81,9 @@ class PanSensorShell(cmd.Cmd):
         Arguments:
             sensor {str} -- environment, weather, webcams
         """
+        # TODO(jamessynge): We currently use a single logger, not one per module or sensor.
+        # Figure out whether to keep this code and make it work, or get rid of it.
```

Comment:
We are not really using the `name` (we call it `profile`) attribute of the logger, which we could start doing. This would allow us to write to one log file but filter based on the name, which could be a module or sensor name.

Reply:
When you say filter, do you mean grep the log file, or filter to decide whether the message is logged. That latter is what I'd like to get out of a change w.r.t. multiple loggers. In particular, it would be nice to be able to enable emitting of debug log messages of selected modules at run time (e.g. by a command-line flag).